rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobResultData.java

```diff
@@ -83,9 +86,18 @@ public void setExceptionResults(Map<String, TaskResultData> exceptionResults) {
         this.exceptionResults = exceptionResults;
     }
 
+    public void setJobMap(Map<String, Serializable> JobMap) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)